ImperatorToCK3.UnitTests/CK3/Titles/LandedTitlesTests.cs

```diff
@@ -309,5 +311,49 @@ public class LandedTitlesTests {
 
 			Assert.Equal(20, titles["c_county1"].GetDevelopmentLevel(date)); // (10+40)/2 - sqrt(25)
 		}
+
+		[Fact]
+		public void DerivedColorsHaveSameHueAsBase() {
+			var titles = new Title.LandedTitles();
+			var baseColor = new Color(0.2, 0.3, 0.4);
+
+			var baseTitle = titles.Add("e_base");
+			baseTitle.Color1 = baseColor;
+
+			var derivedTitle1 = titles.Add("k_derived1");
+			var derivedColor1 = titles.GetDerivedColor(baseColor);
+			derivedTitle1.Color1 = derivedColor1;
+
+			var derivedTitle2 = titles.Add("k_derived2");
+			var derivedColor2 = titles.GetDerivedColor(baseColor);
+			derivedTitle2.Color1 = derivedColor2;
+
+			Assert.Equal(baseColor.H, derivedColor1.H);
```

Comment:
Is there a reason you're not also checking the saturations are the same?

Reply:
No, I can add an assertion.